### DISCUSSION TITLE
in what order do you complete a web project?

### DISCUSSION THREAD

**Discussion by mmoolyaa** [community/community#83097] [Actions]
### Select Topic Area

Show & Tell

### Body

I personally like to finish my design first and continue on from there
Figma (design) -> html, css -> js

feel free to suggest me anything!

**Comment by Shahnawazk7944** [comment]
**_Define requirements -> gather resources -> design -> develop(both front and back end) -> test -> add content -> review ->launch -> maintain._**

**Comment by Hanrryshani** [comment]
Sounds like you're a design-first kinda coder, Figma to code and all that!  Here are some cool tips to keep your workflow smooth

Make your Figma rock: Build interactive prototypes for testing, use a design system for consistency, and don't forget accessibility.
Code conversion magic: Plugins and Figma-to-code tools can be your new best friends. Focus on clean HTML and organized CSS with tools like BEM or utility-first approaches.
JavaScript sprinkles: Start with a basic website and add interactivity with JS later. Frameworks like React or Vue can be handy for complex UIs.
Remember, version control is your BFF, feedback is gold, and experimenting is key! Keep creating and have fun.

**Reply by mmoolyaa** [reply]
Wow I learn a lot of things from this! thanks for giving me your perspective
